examples/Brain/program_rp2/program_rp2.ino

```diff
@@ -0,0 +1,135 @@
+// This sketch program rp2040 by copying UF2 file from SDCard to
+// rp2040 bootrom
+// Hardware wiring:
+// - Brain's Target GPIO28 to RP2 bootsel
+// - Brain's Target Reset to RP2 Reset
+// - Brain's USB host to RP2040 USB interface
+
+// required for Host MSC block device
+#include "SdFat_Adafruit_Fork.h"
+
+// required for USB host
+#include "pio_usb.h"
+#include "Adafruit_TinyUSB.h"
+
+#include "Adafruit_TestBed_Brains.h"
+
+// firmware.h is converted using tools/file2carray.py e.g
+//   python tools/file2carray.py cdc_msc.uf2
+// above command will generate cdc_msc.uf2.h with bindata, bindata_len
+#include "metro_rp2350_cdc_msc.uf2.h"
```

Comment:
@ladyada you can use this new sketch to program rp2350, this use firmware header, which drop the need for SDcard (less thing to worry about). To generate header firmware from uf2 file. Please run `tools/file2carray.py` 

```
python tools/file2carray.py cdc_msc.uf2
```

Reply:
got it!